src/main/java/uk/gov/hmcts/ccd/auditlog/AuditService.java

```diff
@@ -40,7 +40,7 @@ public void audit(AuditContext auditContext) {
         entry.setHttpMethod(auditContext.getHttpMethod());
         entry.setPath(auditContext.getRequestPath());
         entry.setRequestId(auditContext.getRequestId());
-        entry.setIdamId(userRepository.getUser().getEmail());
```

Comment:
maybe let's have our BA Jamal first check with the team who is processing the Audit logs whether this change is ok for them

Reply:
confirmed that this change is ok